tgbot/filters/admin.py

```diff
@@ -0,0 +1,18 @@
+import typing
+
+from aiogram.dispatcher.filters import BoundFilter
+
+from tgbot.config import Config
+
+
+class AdminFilter(BoundFilter):
+    key = 'is_admin'
+
+    def __init__(self, is_admin: typing.Optional[bool] = None):
+        self.is_admin = is_admin
+
+    async def check(self, obj):
+        if self.is_admin is None:
+            return True
+        config: Config = obj.bot.get('config')
```

Comment:
1. Why admin filter knows about all config variables? It should read only his own. 
2. It is hard to customise this filter. For example we cannot read db here. 

That were reasons of splitting role middleware and filter